charts/consul/crds/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
This shouldn't be in here I don't think

Reply:
Oh I see this is being helm ignored. Can you add a comment at the top that this is helm ignored and only used for the make <whatever it is> command?